src/components/ebay-carousel/examples/autoplay.marko

```diff
@@ -1,3 +1,16 @@
+export interface Input {
+    items: number;
+    index?: number;
+    itemsPerSlide?: number;
+    gap?: number;
+    a11yPreviousText?: string;
+    a11yNextText?: string;
+    autoplay?: boolean;
+    paused?: boolean;
+    a11yPauseText?: string;
+    a11yPlayText?: string;
+}
+
```

Comment:
This is probably not needed. Initially we were using another method in storybook to expose variables and this is a remnant of that. I would just simply spread input in the carousel in this file.